### PR TITLE
Function to convert Events in jams.Annotation using segment_open namespace

### PR DESCRIPTION
**Pull request by pzinemanas**
This PR is related to #7. I've created a new `events_to_jams` function which converts `Events` in `jams.Annotation` using [`segment_open`](https://jams.readthedocs.io/en/stable/namespace.html#segment-open) namespace.

### CODE REVIEW

File: soundata/jams_utils.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def jams_converter(
     tags_open_data=None,
     metadata=None,
     tags=None,
+    events=None,
```
Comment (justinsalamon) [inline soundata/jams_utils.py:17]: Plz also add to the docstring :)

File: soundata/jams_utils.py [hunk 2]
```diff
@@ -461,30 +468,30 @@ def tempos_to_jams(tempo_data, description=None):
...
+#     if event_data is not None:
+#         if not isinstance(event_data, annotations.EventData):
+#             raise TypeError("Type should be EventData.")
+#         for beg, end, label in zip(
+#             event_data.intervals[:, 0], event_data.intervals[:, 1], event_data.events
+#         ):
+#             jannot_events.append(time=beg, duration=end - beg, value=label)
+#     if description is not None:
+#         jannot_events.sandbox = jams.Sandbox(name=description)
+#     return jannot_events
```
Comment (justinsalamon) [inline soundata/jams_utils.py]: I guess we can delete this commented out section?

File: tests/test_jams_utils.py [hunk 3]
```diff
@@ -1010,59 +1010,91 @@ def test_tempos():
...
+#     )
+
+#     with pytest.raises(TypeError):
+#         jams_utils.jams_converter(event_data=event_data4)
+#     with pytest.raises(TypeError):
+#         jams_utils.jams_converter(event_data=event_data5)
+#     with pytest.raises(TypeError):
+#         jams_utils.jams_converter(event_data=event_data6)
+
+
```
Comment (justinsalamon) [inline tests/test_jams_utils.py]: remove commented out code?

**Review by justinsalamon — COMMENTED** [review]
@pzinemanas looks great, I think we're mostly just missing a docstring for the `events` argument of the main converter function, and maybe deleting a couple of commented out blocks

**Comment by magdalenafuentes** [comment]
@pzinemanas and @justinsalamon FYI I'm editing this PR to remove unused code and update!

**Review by magdalenafuentes — APPROVED** [review]

File: soundata/jams_utils.py [hunk 4]
```diff
@@ -37,42 +26,10 @@ def jams_converter(
...
-        tags_open_data (list or None):
-            A list of tuples of (str, str), where the first srt is the tag and the second
-            is a descriptor of the annotation.
-        metadata (dict or None):
-            A dictionary containing the clip metadata.
         tags (list or None):
             A list of tuples of (annotations.Tags, str), where the str describes the annotation.
-
+        events (list or None):
+            A list of tuples of (annotations.Events, str), where the str describes the annotation.
```
Comment (pzinemanas) [inline soundata/jams_utils.py]: `events` and `tags` are not list of tuples right now. Instead, these are annotations.Events and annotations.Tags, right @magdalenafuentes ?. Should we accept tuple to include description? Should we expect a list if you have different sets of tags or events?

Comment (magdalenafuentes) [reply soundata/jams_utils.py]: Good catch! You’re right, it should say annotations.Tags and Events. Can you update it?

**Comment by magdalenafuentes** [comment]
@pzinemanas merge at convenience :)

**Pull request merged**